docs/tutorials/app-runtime-mutation.md

```diff
@@ -0,0 +1,242 @@
+---
+id: app-runtime-mutation
+title: Data Mutation  
+---
+
+This tutorial is a continuation of the [DHIS2 application runtime](https://runtime.dhis2.nu/#/getting-started) tutorial on [Data Query](/docs/tutorials/app-runtime-query).  
+ 
+In this section we will:
+
+1. Get started with data mutations 
+2. Learn how to define these mutations
+2. Mutate data using the `useDataMutation` hook
+
+
+### 1. Getting started
+
+### Prerequisites 
+
+Make sure that you have the `@dhis2/app-runtime` dependency installed in your project:
+
+```shell
+yarn add @dhis2/app-runtime
+```
+
+### What are data mutations? 
+
+Data queries are for reading information and **data mutations** are for writing or changing that information. 
+
+With the DHIS2 app runtime, we have 3 different types of mutation which translate directly to REST API requests:
+
+| Mutation Type | REST requests|
+| ------------- |------------- | 
+| Create        | POST         | 
+| Update        | PUT          |   
+| Delete        | DELETE       |    
+
+
+## 2. Define data mutations 
+
+In this example, we will continue building on the application from the [previous tutorial](/docs/tutorials/app-runtime-query) and create a new program by clicking a button. 
+
+To start, let's create a separate file for our `src/NewProgram.js` component which we will import in the `src/App.js`. 
+
+Next, let's define a simple create mutation 👇  
+
+### Create mutation
+
+As with the data query definition from the previous tutorial, the create mutation below is also an object that specifies the resource `programs` from the DHIS2 Web API as well as the following required properties: 
+
+* The property `type` as `create` 
+* The `data` property which is required in type `create` 
+
+:::note 
+The property names of **`data`** must correspond to the property names in the DHIS2 API.
+::: 
+
+```jsx title="src/NewProgram.js"
+// ...
+
+const mutation = {
+    resource: 'programs',
+    type: 'create',
+    data: {
+        name: 'A new Program',
+        shortName: 'A new Program',
+        programType: "WITH_REGISTRATION"
+    }
+}
+// ...
+```
+
+## 3. Mutating data 
+
+We have already declared our data mutation above so now we're ready to use it! 
+
+Let's start by importing the [`useDataQuery` hook](https://runtime.dhis2.nu/#/hooks/useDataMutation) from the `@dhis2/app-runtime`  
+
+### Import `useDataMutation` 
+
+```jsx title="src/NewProgram.js"
+import { useDataQuery } from '@dhis2/app-runtime'
+// ...
```

Comment:
This should be `useDataMutation`, I believe!

Reply:
Yes, of course! 😅  
Thanks Austin for reviewing!